lib/bitcoin/models/ProtocolParameters.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Defines configuration of proof of fee calculation.
+ */
+export default interface ProtocolParameters {
+
+  /** Number of contiguous blocks that go into a group */
+  groupSizeInBlocks: number;
+
+  /** Size of the window in number of groups */
+  windowSizeInGroups: number;
+
+  /**
+   * Transaction fees of bitcoin transactions are rounded so that we need to store
+   * a smaller number of distinct values for computing quantiles. This parameter controls
+   * the rounding - higher the value, poorer the rounded value approximates the original fee,
+   * but lesser the space.
+   */
+  feeApproximation: number;
```

Comment:
I don't think this is needed to be expose in v1.

Reply:
Will make the change in the subsequent integration changes with the core.